src/applications/mhv/medical-records/containers/HealthConditions.jsx

```diff
@@ -30,7 +57,14 @@ const HealthConditions = () => {
     [dispatch],
   );
 
+  const accessAlert = activeAlert && activeAlert.type === ALERT_TYPE_ERROR;
+
   const content = () => {
+    if (accessAlert) {
+      return (
+        <AccessTroubleAlertBox alertType={accessAlertTypes.HEALTH_CONDITIONS} />
+      );
+    }
```

Comment:
The error case lacks an `<h1>` tag. See how this is handled in AllergyDetails and VaccineDetails.

Reply:
The list pages and details pages are set up differently because of the relevant data needed to set up the h1 tag. An h1 tag is always present for list pages so it is not returned as part of the error content.

Reply:
Copy that!